Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM python:3.7
 RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
 
 RUN apt-get update && apt-get --no-install-recommends install --yes \
-    libaio1 libaio-dev xmlsec1 libffi-dev \
-    libldap2-dev libsasl2-dev \
+    libaio1 libaio-dev xmlsec1 libffi-dev libsasl2-dev \
```

Comment:
Looks like we dropped `libldap2` as well as `python-ldap`. Can you explain briefly the removal? I'm not familiar with LDAP.

Reply:
I don't think it should matter 3.7 or 3.8. 3.7 is supported still till [June 2023. ](https://endoflife.date/python). I feel like this app would be shut off by then. I can switch to 3.8 and see if there's any issues. Usually updating Django involves more work it seems than Python.

Reply:
I removed LDAP when @pushyamig noticed it wasn't used. I looked at the history and saw it was used early in the project and just never removed from the requirements. I don't see it anywhere in the code.

It's possible it was used long before me in 2016 when SAML support was added. This was a common usage to authenticate. I don't see it in the history what it was actually used for.

I know we're not using it for anything now. We have a special attribute in SAML to indicate isMemberOf for group authentication, and it could have also used LDAP for that but I didn't see usage of that either. 

Just has been unused and works fine without it!

Reply:
Cool, thanks for explanation

Reply:
I don't expect there to be an issues with 3.8. I'd probably just bump.

Reply:
@jonespm can we just bump to 3.8 and test accordingly?

Reply:
I switched to 3.8-slim and only had to add npm to install. I ran the tests which all passed so looks good!